### ISSUE TITLE
PC Cam shows but Avatarify Cam does not display

### ISSUE THREAD

**Issue by BigBLAZE** [alievk/avatarify-python#444]
I had a difficult time getting Avatarify to work on my PC. I have a MSI Katana GF76 11UD 11th Gen Intel(R) Core(TM) i7-11800H @ 2.30GHz   2.30 GHz with 16gb Ram and Nvidia GeForce RTX 3050 Ti Laptup GPU. Eventually I got it to work but only my cam shows up and the Avatarify cam does not show up. Before I was able to get the Scripts to run successfully, the initial error was

 '"OMP: Error #15: Initializing libiomp5md.dll, but found libiomp5md.dll already initialized.
OMP: Hint This means that multiple copies of the OpenMP runtime have been linked into the program. That is dangerous, since it can degrade performance or cause incorrect results. The best thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupported, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce incorrect results. For more information, please see http://www.intel.com/software/products/support/.'"

so in order to solve this, I searched for all instances of "libiomp5md.dll" and deleted only one from "C:\ProgramData\Miniconda3\envs\avatarify\Library\bin" -after which the scripts ran but here the problem of only my cam showed up. 

Please I need some insight on this!

**Comment by BigBLAZE** [comment]
C:\ProgramData\Miniconda3\envs\avatarify\lib\site-packages\torch\functional.py:568: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at  C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\TensorShape.cpp:2228.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]



this is what I get after "run_windows.bat" command

**Comment by BigBLAZE** [comment]
now both windows show up but it gets very slow and the lag is unbelievable. could this be my GPU or my CPU not being able to handle the process? Do I need another PC with even higher specs than what I have now?

**Comment by BigBLAZE** [comment]
this is an update as to the current slowness. The message displayed while both cams are on is quoted below

"C:\ProgramData\Miniconda3\envs\avatarify\lib\site-packages\torch\nn\functional.py:4194: UserWarning: Default grid_sample and affine_grid behavior has changed to align_corners=False since 1.3.0. Please specify align_corners=True if the old behavior is desired. See the documentation of grid_sample for details.
  "Default grid_sample and affine_grid behavior has changed "
C:\ProgramData\Miniconda3\envs\avatarify\lib\site-packages\torch\nn\functional.py:1944: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")"


Please help!

**Comment by JohanAR** [comment]
Your hardware is more than enough, but it might not be using the GPU at all. I'm guessing some dependencies (Torch / CUDA) needs to be uplifted

**Comment by Orebab** [comment]
> now both windows show up but it gets very slow and the lag is unbelievable. could this be my GPU or my CPU not being able to handle the process? Do I need another PC with even higher specs than what I have now?

I am having similar issues how do i fix it so that both cam can work

**Comment by Orebab** [comment]
C:\Users\Oreva\miniconda3\envs\avatarify\lib\site-packages\torch\nn\functional.py:3103: UserWarning: The default behavior for interpolate/upsample with float scale_factor changed in 1.6.0 to align with other frameworks/libraries, and now uses scale_factor directly, instead of relying on the computed output size. If you wish to restore the old behavior, please set recompute_scale_factor=True. See the documentation of nn.Upsample for details.  this what i get after run_windows.bat only one cam work avatariry does not come up

**Comment by 12ayfa** [comment]
> this is an update as to the current slowness. The message displayed while both cams are on is quoted below
> 
> "C:\ProgramData\Miniconda3\envs\avatarify\lib\site-packages\torch\nn\functional.py:4194: UserWarning: Default grid_sample and affine_grid behavior has changed to align_corners=False since 1.3.0. Please specify align_corners=True if the old behavior is desired. See the documentation of grid_sample for details. "Default grid_sample and affine_grid behavior has changed " C:\ProgramData\Miniconda3\envs\avatarify\lib\site-packages\torch\nn\functional.py:1944: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead. warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")"
> 
> Please help!

Bro mine does thesame and shows same warnings 
Were you able to fix it?? And how 
Pls help 🙏
